src/redux/store.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './book/bookSlice';
+import categoryReducer from './category/categorySlice';
+
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] It would be better if you don't disable the linters check and manually fix the linters because linter prevent you from making those error which can break your application 😨 You can fix this linter check by doing something which i did below ⬇️


```javascript
const store = configureStore({
  reducer: {
    book: bookReducer,
    category: categoryReducer,
  },
});

export default store;

```

- [ ] After exporting it as default you will need to import it as below 👇

**🟢 Like This 👍**
```
import store from 'put here the path'
```

**🔴 Not Like This 👎**
```
import { store } from 'put here the path'
```

Reply:
thanks I will make the changes bro